lesson25/index.html

```diff
@@ -4,7 +4,8 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <script src="./app.js" defer></script>
+    <script type="module" src="./js/app.js" defer></script>
+    <script type="module" src="./js/validation.js" defer></script>
```

Comment:
nits
> There is no need to use the defer attribute (see [<script> attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#attributes)) when loading a module script; modules are deferred automatically.
```suggestion
    <script type="module" src="./js/app.js"></script>
    <script type="module" src="./js/validation.js"></script>
```
[ref](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules#other_differences_between_modules_and_standard_scripts)

Reply:
You pointed this out last time. Sorry for repeating myself! I have corrected it.
aba6dbf